src/wkt/tests/message_with_bool.rs

```diff
@@ -41,6 +41,7 @@ mod test {
     #[test_case(MessageWithBool::new().set_or_clear_optional(None::<bool>), json!({}))]
     #[test_case(MessageWithBool::new().set_repeated([true, true, false]), json!({"repeated": [true, true, false]}))]
     #[test_case(MessageWithBool::new().set_map_key([(true, "trueValue"), (false, "falseValue")]), json!({"mapKey": {"true": "trueValue", "false": "falseValue"}}))]
+    #[test_case(MessageWithBool::new().set_map_key([(true, "trueValue")]), json!({"mapKey": {"tr\u{0075}e": "trueValue"}}))]
```

Comment:
let's use 'u' instead of the unicode for 'u'. :slightly_smiling_face:

Reply:
That is the point... we are trying to verify the unicode for `u` deserializes correctly.

Reply:
Can you name the test case then? `; "handles unicode in key"`

I was obviously confused by `tr\u{0075}e`, the next person may be too.